packages/components/src/presentation-components/properties/NumericPropertyEditor.tsx

```diff
@@ -0,0 +1,74 @@
+import { createRef, PureComponent } from 'react';
+
+import { StandardTypeNames } from '@itwin/appui-abstract';
```

Comment:
Imports sorted  different than in other files. Could you copy import sorter settings from `presentation/.vscode/settings.json` to your workspace if you are using `CoSpace` and rerun import sorter?

Reply:
fixed?